save_changes.py

```diff
@@ -6,28 +6,42 @@
 import os
 import subprocess
 import sys
+from os import path
```

Comment:
Please clean up this import. There are some references using `os.path` and `path`. I think we should revert back to just having `import os`, but making the changes to import each function individually would be agreeable as well.